public_html/wp-content/plugins/wordcamp-attendee-survey/tests/test-wordcamp-attendee-survey.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace WordCamp\AttendeeSurvey\Tests;
+
+use WP_UnitTestCase;
+
+defined( 'WPINC' ) || die();
+
+/**
+ * Class Test_AttendeeSurvey
+ *
+ * @group wordcamp-attendee-survey
+ */
+class Test_AttendeeSurvey extends WP_UnitTestCase {
```

Comment:
I setup this test framework but to be honest, there aren't many pure functions worth testing. I think I'll pull it out before merge. Agree/Disagree?

Reply:
It's fairly typical IME to not have a lot of pure functions in WP code. Most of our PHP tests are really _integration_ tests rather than _unit_.

It's still possible to test things if you want, though. Here's a couple examples:

* https://github.com/WordPress/wordcamp.org/blob/ca23915cea8ef2d4509027e068f67aa47f16cc86/public_html/wp-content/plugins/wordcamp-organizer-reminders/tests/test-wcor-mailer.php
* https://github.com/WordPress/two-factor/pull/587/files

In our context, though, I don't think automated tests are usually worth the time involvement, unless it's something dealing with security, something that's cumbersome to manually test, prone to breaking, etc.

Reply:
Yeah, I added the test suite before starting and didn't find anything testable :). Integration tests make sense but not unit tests for this PR. I'll remove.